ColumbiaRoad/impact-helper-api
# Feature/set up redis

Set up a Redis cache and create an example server method with caching.

## How to test

- Run [Redis](https://redis.io/docs/getting-started/) locally on the default port (6379).
- Run the Hapi server and go to `/example` on your browser.
- [ ] You should see a timestamp.
- [ ] When you refresh the page within the next 10 seconds, the timestamp should remain the same.
- [ ] After 10 seconds, when you refresh the page, the timestamp should be updated.
